homeassistant/components/climate/__init__.py

```diff
@@ -43,6 +43,9 @@
 STATE_AUTO = "auto"
 STATE_DRY = "dry"
 STATE_FAN_ONLY = "fan_only"
+STATE_AWAY_HOLD = "away"
+STATE_HOME_HOLD = "home"
+STATE_TEMPERATURE_HOLD = "temp"
```

Comment:
In terms of the ecobee, "home", and "away" are "climates", which define various parameters on the thermostat. I don't know what they mean on other thermostats.

I also don't know if it makes sense to define strings that are currently only being used by one thermostat (ecobee) in __init__. Will they be re-used in other thermostats?